HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc

```diff
@@ -27,11 +29,14 @@ HLTBTagHarvestingAnalyzer::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGet
 	{
 		dqmFolder_hist = Form("HLT/BTag/Discriminator/%s",hltPathNames_[ind].c_str());
 		std::string effDir = Form("HLT/BTag/Discriminator/%s/efficiency",hltPathNames_[ind].c_str());
+		std::string relationsDir = Form("HLT/BTag/Discriminator/%s/HEP17_HEM17",hltPathNames_[ind].c_str());
 		ibooker.setCurrentFolder(effDir);
 		TH1 *den =NULL;
 		TH1 *num =NULL; 
 		std::map<TString,TH1F> effics;
 		std::map<TString,bool> efficsOK;
+		std::map<TString,std::map<std::string,TH1F> > efficsmod;
```

Comment:
@fwyzard OK, I will. Then I would leave the current one as is and introduce the mentioned improvements (cloned histograms and TString) in a separate PR.